databuilder/extractor/hive_table_metadata_extractor.py

```diff
@@ -66,9 +67,13 @@ def init(self, conf):
         conf = conf.with_fallback(HiveTableMetadataExtractor.DEFAULT_CONFIG)
         self._cluster = '{}'.format(conf.get_string(HiveTableMetadataExtractor.CLUSTER_KEY))
 
-        self.sql_stmt = HiveTableMetadataExtractor.SQL_STATEMENT.format(
+        default_sql = HiveTableMetadataExtractor.DEFAULT_SQL_STATEMENT.format(
             where_clause_suffix=conf.get_string(HiveTableMetadataExtractor.WHERE_CLAUSE_SUFFIX_KEY))
 
+        self.sql_stmt = conf.get_string(HiveTableMetadataExtractor.EXTRACT_SQL.format(
```

Comment:
I think it's missing closing bracket?

```
conf.get_string(HiveTableMetadataExtractor.EXTRACT_SQL.format
--> conf.get_string(HiveTableMetadataExtractor.EXTRACT_SQL).format

```

By the way, we may not need to add where clause if they provide SQL statement. WDYT?

Reply:
I think if they don't provide the where statement, it will just use the same SQL provided by user. Also I think the bracket should work (it is in the end of the statement) as I have an additional test.

Reply:
actually yeah, we should just let them provide the sql.